feedthefox/users/adapters.py

```diff
@@ -36,8 +36,18 @@ def populate_user(self, request, sociallogin, data):
         user = super(FeedTheFoxSocialAdapter, self).populate_user(request,
                                                                   sociallogin,
                                                                   data)
+        query_email = user.email
+
+        if sociallogin.account.provider == 'github':
+            email_addresses = sociallogin.account.extra_data.get('email_addresses')
+            verified_address = [address['email'] for address in email_addresses
```

Comment:
What is missing from here is to check against all the possible emails returned from github, thus the list. This is already tracked in an issue. I am going to try to fix this in this PR.